share/src/main/java/io/zensoft/share/listener/Facebook/FacebookListener.java

```diff
@@ -0,0 +1,33 @@
+package io.zensoft.share.listener.Facebook;
+
+import io.zensoft.share.config.rabbitmq.RabbitMqComponentDeclarationConfiguration;
+import io.zensoft.share.config.rabbitmq.RabbitMqExchangeBindingConfiguration;
+import io.zensoft.share.dto.VacancyDto;
+import io.zensoft.share.listener.VacancyListener;
+import io.zensoft.share.service.PublisherManagerService;
+import io.zensoft.share.service.converter.DefaultVacancyConverterService;
+import io.zensoft.share.service.converter.DefaultVacancyResponseConverterService;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class FacebookListener implements VacancyListener {
+
+    private final PublisherManagerService facebookPublisherManagerService;
+
+    public FacebookListener(PublisherManagerService facebookPublisherManagerService){
```

Comment:
if you have constructor, i guess it should annotated with `@Autowired` annotation